content/influxdb/v2.1/upgrade/v2-to-v2.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Upgrade from InfluxDB 2.x to InfluxDB 2.1
+description: >
+  To upgrade from InfluxDB 2.0 beta 16 or earlier to InfluxDB 2.1 (stable),
+  manually upgrade all resources and data to the latest version by completing these steps.
+menu:
+  influxdb_2_1:
+    parent: Upgrade InfluxDB
+    name: InfluxDB 2.x to 2.1
+weight: 10
+related:
+  - /influxdb/v2.1/reference/cli/influxd/downgrade/
+  - /influxdb/v2.1/install/
+---
+
+Upgrade to InfluxDB {{< current-version >}} from an earlier version of InfluxDB 2.x.
+
+{{% note %}}
+#### InfluxDB 2.0 beta-16 or earlier
+If you're upgrading from InfluxDB 2.0 beta-16 or earlier, you must first [upgrade to InfluxDB 2.0](http://localhost:1313/influxdb/v2.0/upgrade/v2-beta-to-v2/), 
```

Comment:
Should `http://localhost:1313` be replaced by something else in the link here?

Reply:
Yes 🤦🏻 thank you.